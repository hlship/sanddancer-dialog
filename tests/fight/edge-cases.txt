(setup chase)
take jade
n
w
(now) (#chase has completed)
z
brood stories
x coyotes
s
throw jade at coyotes