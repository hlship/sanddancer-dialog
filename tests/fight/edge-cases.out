

Middle of Nowhere (in the pickup truck)
The tire tracks from the south stop abruptly here, but where the hell are you?
The desert sand and clumps of pale sagebrush are all your dimming headlights
pick out before barely reaching the concrete building to the north.

Your poor old pickup truck ticks and groans, smashed gracelessly against a tall
Saguaro; wiffs of evaporating gasoline linger in the chill air. On the dash you
can see a piece of jade. The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> (setup chase)
Query succeeded: (setup chase)
> take jade
You take the piece of jade.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> n
(first attempting to get out of the pickup truck)
You work the rusty truck door open. You get out of the pickup truck.

You walk north.

Crumbling Concrete
This building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

> w
You walk west for a long time, thoughts turning over like the sand under your
sneakers. You think about your grandma's stories. You think about Ocean and how
pissed she's gonna be that you're not home. Like you tell her not to wait up for
you but she always does anyway.

Some wind kicks up and chucks sand in your face. You blink it out and kinda
realize while you're doing it that you haven't really been paying attention to
where you've been going. Another gust whines through the sagebrush and you
realize with a kind of heavy feeling that with all the dust this wind is kicking
up you can't see a damn thing. You can't see your pickup, you can't see the
electrical tower. Not even your damn footprints. You're lost. Well, shit.

Open Desert
Clouds of dust swirl through the air, and you shiver in the chill night air.

You can sort of see a looming cactus, a dry streambed, and some clumps of
sagebrush.

> (now) (#chase has completed)
> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

It's over. They've caught you. They surround you, circling, growling, looking
for weakness, moving in for the kill.

> brood stories
No time for that now.

The circle of snarling coyotes tightens.

> x coyotes
They're all around you, teeth and fur and sweat and muscles, angry, closing.

The coyotes growl, pulling closer, snapping at your heels.

> s
You try to flee and instantly teeth and claws block your path; you pull back to
the center of the circle, terrified.

The circle of snarling coyotes tightens.

> throw jade at coyotes
Maybe you could just attack them directly.

A coyote snaps his teeth, lunging foward, then fading back into the pack.

> 
